app/controllers/questions/filing_for_stimulus_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Questions
+  class FilingForStimulusController < QuestionsController
+    layout "yes_no_question"
+
+    def next_path
+      if current_intake.filing_for_stimulus_yes? && current_intake.already_filed_yes?
+        stimulus_recommendation_path
+      else
+        super
+      end
+    end
+
+    def track_question_answer
```

Comment:
This seemed like the easiest way to fill Lou's suggestion: "we should log responses to this in mixpanel as a custom event so we can easily report on what % of people are electing stimulus". If it is important to have the usual "question_answered" event as well then I could just call super above.

Reply:
`track_question_answer` is probably sufficient for this